openstack_project_manager/manage.py

```diff
@@ -375,6 +375,47 @@ def check_volume_types(
                 pass
 
 
+def manage_private_volumetypes(
+    configuration: Configuration,
+    project: openstack.identity.v3.project.Project,
+    domain: openstack.identity.v3.domain.Domain,
+) -> None:
+    admin_project = configuration.os_cloud.get_project(
+        name_or_id="admin", domain_id="default"
+    )
+
+    if not admin_project or project.id == admin_project.id:
+        return
+
+    logger.info(
+        f"{project.name} - Applying private volume types for domain {domain.name}"
+    )
+
+    all_volume_types = list(configuration.os_cloud.block_storage.types(is_public=False))
+
+    for volume_type in all_volume_types:
+        if not volume_type.name.startswith(f"{domain.name}-"):
```

Comment:
Use upper() here. The domain name can be e.g. osism or OSISM or OsIsM, the volume type will always be e.g. OSISM-XXX.